docs/dev/stream/operators/windows.md

```diff
@@ -460,118 +465,14 @@ The above example appends all input `Long` values to an initially empty `String`
 
 <span class="label label-danger">Attention</span> `fold()` cannot be used with session windows or other mergeable windows.
 
-### WindowFunction - The Generic Case
+### ProcessWindowFunction
 
-A `WindowFunction` gets an `Iterable` containing all the elements of the window and provides
+A `ProcessWindowFunction` gets an `Iterable` containing all the elements of the window and provides
 the most flexibility of all window functions. This comes
```

Comment:
A `ProcessWindowFunction` gets an `Iterable` containing all the elements of the window, and a `Context` object with access to time and state information, which enables it to provide more flexibility than other window functions.

Reply:
Thanks, this sounds better!